app/main.py

```diff
@@ -1,3 +1,62 @@
+import functools
+from typing import Callable, Any
+
+
+def validate_operand(func: Callable) -> Callable:
+    @functools.wraps(func)
+    def wrapper(self: "Distance", other: ("Distance", int, float)) -> Any:
```

Comment:
You could omit quotes on DIstance type if you use annotations from __future__

Reply:
Resolved